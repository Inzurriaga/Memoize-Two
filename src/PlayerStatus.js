import React from "react"

export default function PlayerStatus(props){
    return(
        <div className="status-section">
            <h2>{props.user.name.toUpperCase()}</h2>
        </div>
    )
}