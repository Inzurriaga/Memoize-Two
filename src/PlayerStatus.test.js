import React from 'react';
import { shallow } from 'enzyme';
import PlayerStatus from "./PlayerStatus"

const mockUser = {"name":"gabriel","password":"gabriel","completed":[1,2,5]}

describe("PlayerStatus", ()=> {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(
            <PlayerStatus user={mockUser}/>
        )
    })

    it("shold match the snapshot", () => {
        expect(wrapper).toMatchSnapshot()
    })
})