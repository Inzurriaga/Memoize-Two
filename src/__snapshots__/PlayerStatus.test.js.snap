// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerStatus shold match the snapshot 1`] = `
<div
  className="status-section"
>
  <h2>
    GABRIEL
  </h2>
</div>
`;
